src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -77,7 +77,18 @@ private void ScanStringLiteral(ref TokenInfo info, bool inDirective)
             }
             else
             {
-                info.Kind = SyntaxKind.StringLiteralToken;
+                if (!inDirective && TextWindow.PeekChar() == 'u' && TextWindow.PeekChar(1) == '8')
```

Comment:
nit: this may follow the spec, but we probably want to take back to ldm if we support `U8` as well as `u8` all our other suffixes are case insensitive. so we probably want the same here.

Reply:
>this may follow the spec, but we probably want to take back to ldm if we support ```U8``` as well as ```u8``` all our other suffixes are case insensitive. so we probably want the same here.

Will add a PROTOTYPE comment and will bring this design question to LDM